servers/dsub/jobs/test/base_test_cases.py

```diff
@@ -548,94 +499,94 @@ def test_get_job_aggregations(self):
-                sleep(10)
+            # wait_status need 10s on local provider
+            if current_app.config['PROVIDER_TYPE'] == ProviderType.LOCAL:
+                time.sleep(10)
 
             self.wait_status(job_aborted, ApiStatus.RUNNING)
             self.must_abort_job(job_aborted)
-
             self.wait_status(job_aborted, ApiStatus.ABORTED)
 
             self.wait_status(job_failed, ApiStatus.FAILED)
 
             self.wait_status(job_succeeded, ApiStatus.SUCCEEDED)
 
             self.wait_status(job_running, ApiStatus.RUNNING)
-            self.wait_status(job_running_no_token, ApiStatus.RUNNING)
-            self.wait_status(job_running_no_token_diff_label,
-                             ApiStatus.RUNNING)
             self.wait_status(job_running_diff_label_value, ApiStatus.RUNNING)
             self.wait_status(job_running_diff_labels, ApiStatus.RUNNING)
 
             userId = self.must_get_job(job_running).extensions.user_id
             aggregation_resp = self.must_get_job_aggregations('HOURS_1')
 
-            self.assert_aggregation_response_equal(aggregation_resp, userId,
-                                                   name)
+            aggregation_response_dict = self.aggregation_response_to_dict(
+                aggregation_resp)
+
+            status_counts_total = {
+                'Running': 3,
+                'Failed': 1,
+                'Succeeded': 1,
+                'Aborted': 1
+            }
+
+            status_counts_group = {
+                'Running': 1,
+                'Failed': 1,
+                'Succeeded': 1,
+                'Aborted': 1
+            }
+
+            status_counts_single = {'Running': 1}
+
+            expected_aggregations_response_dict = {
+                'summary': status_counts_total,
+                'userId': {
+                    userId: status_counts_total
+                },
+                'name': {
+                    name: status_counts_total
+                },
+                'aggregation-testing-unique-1': {
+                    'testing-rocks': status_counts_group,
+                    'different-value': status_counts_single
+                },
+                'test_token': {
+                    self.test_token_label['test_token']: status_counts_total,
+                },
+                'different-label': {
+                    'different-value': status_counts_single
+                }
+            }
+
+            self.assertEqual(aggregation_response_dict,
+                             expected_aggregations_response_dict)
 
             # Cancel jobs for testing
-            self.must_abort_job(job_running)
-            self.must_abort_job(job_running_no_token)
-            self.must_abort_job(job_running_diff_labels)
-            self.must_abort_job(job_running_diff_label_value)
-            self.must_abort_job(job_running_no_token_diff_label)
+            self.client.open(
```

Comment:
`for j in [job_running, job_running...]:`

Alternatively, accumulate in a list per above comment